sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -315,6 +337,97 @@ def get_speed_lane_count_and_subport(port, cfg_port_tbl):
     return port_speed, lane_count, subport_num
 
 
+def convert_to_int32(value):
```

Comment:
@longhuan-cisco I am not sure why custom values need this conversion compared to existing SI values